cookiecutter.json

```diff
@@ -23,4 +24,4 @@
         "GNU LGPL v3.0",
         "GNU GPL v3.0"
     ]
-}
+}
```

Comment:
@DragaDoncila can you change this setting in VSCode? I just did it in mine. The diffs have been driving me nuts but I finally just looked it up :joy: 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files